example/spec/parallel/emsReport/emsIngestDistributionReportSpec.js

```diff
@@ -237,4 +264,85 @@ describe('The EMS report', () => {
       results.forEach((result) => expect(result).not.toBe(false));
     });
   });
+
+  describe('When there are distribution requests', () => {
+    let accessToken;
+
+    beforeAll(async () => {
+      setDistributionApiEnvVars();
+      const accessTokenResponse = await getEarthdataAccessToken({
+        redirectUri: process.env.DISTRIBUTION_REDIRECT_ENDPOINT,
+        requestOrigin: process.env.DISTRIBUTION_ENDPOINT
+      });
+      accessToken = accessTokenResponse.accessToken;
+    });
+
+    afterAll(async () => {
+      await accessTokensModel.delete({ accessToken });
+    });
+
+    // the s3 server access log records are delivered within a few hours of the time that they are recorded,
```

Comment:
So then if I access a file, generate the reports and it doesn't include the file, will that file be included in the next day's report?

Reply:
Yes.  Also, if the granule of the file is still in the database (this test leave 2 granules in the table), the flat file includes additional information (collection, granuleId and file type).

Reply:
To extend a little on Lauren's question. Whether or not access logs are included in a report is dependent on the updated time of the log report in s3, correct? That should ensure that we don't ever miss logs from one day to another.

Reply:
Yes.  Since there are so many access logs, only the access logs after the `startTime` of the EMS distribute report are processed, the event time in the access log is compared to the `startTime` to further narrow down the distribution request.